pages/blog/_posts/2020-01-28-Thanos-infinite-metrics.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blogpost
+permalink: /blog/metrics-unlimited-thanos/
+title: "Towards Observability nirvana: Infinite metric retention with Thanos"
```

Comment:
Either more capitals or less (I prefer less) for the title :)

Reply:
Done